.github/workflows/lintdiff.yaml

```
@@ -0,0 +1,68 @@
+name: LintDiff (preview)
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  lintdiff:
+    name: LintDiff (preview)
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout eng
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            eng/
+            .github/
+
+      - name: Checkout 'after' state
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          path: after
+
+      - name: Checkout 'before' state
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+          path: before
+
+      - name: Setup Node and install deps
+        uses: ./.github/actions/setup-node-install-deps
+        with:
+          working-directory: eng/tools/lint-diff
+          # TODO: this probably needs a package-lock.json and dist
+          install-command: npm i
+      - run: npm run build
+        working-directory: eng/tools/lint-diff
+
+      # TODO: This can probably be updated to run directly from JS
+      # TODO: default workspace is the after/ folder
+      - name: Get changed files
+        run: |
+          ."./eng/scripts/ChangedFiles-Functions.ps1"
+          Set-Location ./after
+          Get-ChangedFiles | Set-Content -Path ../changed-files.txt
+          Get-Location
+          Get-Content ../changed-files.txt
+        shell: pwsh
+
+      - run: |
+          pwd
+          ls -lh
+
+      - name: Run LintDiff Before
+        run: |
+          realpath ../../../before
+          realpath ../../../changed-files.txt
+          npm exec -- lint-diff --path ../../../before --changed-files-path ../../../changed-files.txt --out-file placeholder.txt
```

```suggestion
          npm exec --no -- lint-diff --path ../../../before --changed-files-path ../../../changed-files.txt --out-file placeholder.txt
```

Add param `--no` to ensure only pre-installed packages are used.